setup.py

```diff
@@ -32,7 +33,7 @@
     keywords='aircraft design optimization multidisciplinary multi-disciplinary analysis',
     author='Benjamin J. Brelje and Eytan J. Adler',
     author_email='bbrelje@umich.edu and eytana@umich.edu',
```

Comment:
Maybe you wanna get rid of the emails here, unless you wanna get random emails from people. I personally think the only way to contact the devs should be on GitHub

Reply:
Ok, will do